### PR TITLE
feat: pass client version

### PR DESCRIPTION
**Pull request by shuplenkov**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ class DAPIClient {
     const urlToConnect = await this.getGrpcUrl();
 
     const client = new CorePromiseClient(urlToConnect);
-    client.setProtocolVersion(version);
+    client.setProtocolVersion(protocolVersion);
```
Comment (shumkov) [inline src/index.js:152]: Should be a string, not an integer?

Comment (jawid-h) [reply src/index.js:152]: @shumkov `package.json` would return string isn't it?

Comment (shuplenkov) [reply src/index.js:152]: And we use semver to check it

Comment (shumkov) [reply src/index.js:152]: Yeah, we changed this part in the common lib. It's good now.

File: src/index.js [hunk 2]
```diff
@@ -339,7 +345,7 @@ class DAPIClient {
 
     const urlToConnect = await this.getGrpcUrl();
 
-    const client = new TransactionsFilterStreamPromiseClient(urlToConnect);
+    const client = new TransactionsFilterStreamPromiseClient(urlToConnect, protocolVersion);
```
Comment (shumkov) [inline src/index.js:348]: Through setter now?

**Pull request closed without merging**